azure-pipelines-official.yml

```diff
@@ -71,8 +71,8 @@ stages:
     displayName: Official Build
     timeoutInMinutes: 360
     pool:
-      name: NetCoreInternal-Pool
-      queue: BuildPool.Server.Amd64.VS2017
+      name: NetCore1ESPool-Svc-Internal
```

Comment:
Can we move the pool up to the build stage? This way it would also apply to the "Publish to Build Asset Registry" Job that does not specify a build pool.

Reply:
Sure!